js/site-nav.js

```diff
@@ -65,6 +67,14 @@ SiteNav.prototype.initMegaMenu = function() {
       focusClass: 'is-focus',
       openClass: 'is-open'
     });
+
+    new typeahead.Typeahead('.js-menu-search', 'candidates', '/data/');
+  }
+};
+
+SiteNav.prototype.destroyMegaMenu = function() {
+  if ( $('body').width() < helpers.BREAKPOINTS.LARGE ) {
```

Comment:
Nope. The library we're using doesn't have a `destroy()` method, so bringing it back re-initailizes it all together and rebinds all the event handlers and everything. I think it's ok though. It's more of a temporary fix until we get the mobile version going.